lib/athenaExpress.js

```diff
@@ -11,17 +11,9 @@ module.exports = class AthenaExpress {
     constructor(init) {
         helpers.validateConstructor(init);
         this.config = {
-            athena: new init.aws.Athena({
-                apiVersion: "2017-05-18",
-            }),
-            s3: new init.aws.S3({
-                apiVersion: "2006-03-01",
-            }),
-            s3Bucket:
-                init.s3 ||
-                `s3://athena-express-${init.aws.config.credentials.accessKeyId
```

Comment:
no longer creates a bucket. This would have been a bug anyway, if user code didn't supply `s3` nor access credentials.